DontPanicLabs.Ifx.Configuration.Local/README.md

```diff
@@ -1 +1,94 @@
-# DontPanicLabs.Ifx.Configuration.Local
\ No newline at end of file
+This project supports three configuration sources:
```

Comment:
You need a single top level `# H1`, then your sections need to be `## H2`. An `#` will turn into a `<h1>` (`##` to `<h2>`, `###` to `<h3>`). For accessibility and semantic HTML reasons, it's good practice to only have a single `<h1>` header for a _thing_. In this instance, I believe README is rendered into is an `<article></article>`, the focal point of the page.